src/services/freshchatService.js

```
@@ -1,243 +1,282 @@
+
+        console.log("Total users fetched:", response.data);
+
+        //STORE USERS'S ids
+        for (const user of users) {
+          userIds.push(user.id);
         }
 
-        return userIds;
-    }
+        let currentPage = response.data.pagination.current_page;
+        let totalPages = response.data.pagination.total_pages;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'totalPages' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)